src/collection/queries/get-collections-by-owner.graphql

```diff
@@ -1,5 +1,5 @@
 query getCollectionsByOwner(
```

Comment:
also update the query name and the file name so it is more clear that the collaborator id that is passed can be an owner or a contributor
getCollectionsByOwnerOrContributor